src/sentry/tasks/statistical_detectors.py

```diff
@@ -216,7 +216,7 @@ def _detect_transaction_change_points(
         request = {
             "data": data,
             "sort": "-trend_percentage()",
-            "trendFunction": trend_function,
+            "min_change()": 100_000_000,  # require a minimum 100ms increase (in ns)
```

Comment:
won't you still need to pass `trendFunction`?

Reply:
i checked but that doesnt do anything